NodeBase/entry_point.sh

```diff
@@ -32,10 +32,13 @@ fi
 # TODO: Look into http://www.seleniumhq.org/docs/05_selenium_rc.jsp#browser-side-logs
 
 SERVERNUM=$(get_server_num)
+
+rm -f /tmp/.X*lock
```

Comment:
@addteq-himanshu I also added this change from https://github.com/SeleniumHQ/docker-selenium/pull/230 as I experienced this behavior when restarting containers.